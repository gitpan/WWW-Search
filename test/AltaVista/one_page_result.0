<html><head>
<title>AltaVista: Simple Query +LSAM +AutoSearch</title>
</head><body>
<a href="/cgi-bin/query?pg=q&text=yes"><b>AltaVista Search</b></a><br>
<FORM name=mfrm method=GET action="/cgi-bin/query">
<INPUT TYPE=hidden NAME=pg VALUE=q>
<INPUT TYPE=hidden NAME=text VALUE=yes>
<font size=-1 face="arial, helvetica"><B>Search <SELECT NAME=what><OPTION VALUE=web SELECTED>the Web<OPTION VALUE=news >Usenet</SELECT>
for documents in <SELECT NAME=kl><OPTION VALUE=XX SELECTED>any language 
<OPTION VALUE=zh >Chinese
<OPTION VALUE=cz >Czech
<OPTION VALUE=da >Danish
<OPTION VALUE=nl >Dutch
<OPTION VALUE=en >English
<OPTION VALUE=et >Estonian
<OPTION VALUE=fi >Finnish
<OPTION VALUE=fr >French
<OPTION VALUE=de >German
<OPTION VALUE=el >Greek
<OPTION VALUE=he >Hebrew
<OPTION VALUE=hu >Hungarian
<OPTION VALUE=is >Icelandic
<OPTION VALUE=it >Italian
<OPTION VALUE=ja >Japanese
<OPTION VALUE=ko >Korean
<OPTION VALUE=lv >Latvian
<OPTION VALUE=lt >Lithuanian
<OPTION VALUE=no >Norwegian
<OPTION VALUE=pl >Polish
<OPTION VALUE=pt >Portuguese
<OPTION VALUE=ro >Romanian
<OPTION VALUE=ru >Russian
<OPTION VALUE=es >Spanish
<OPTION VALUE=sv >Swedish
</SELECT></B></font><br><INPUT NAME=q size=65 maxlength=800 wrap=virtual VALUE="+LSAM +AutoSearch"><br><INPUT TYPE=radio NAME=act VALUE=search CHECKED>search <INPUT TYPE=radio NAME=act VALUE=refine>refine <INPUT TYPE=submit VALUE=Submit>
 <BR><a href="/av/content/help_simple.htm">Help</a> . <a href="/cgi-bin/query?pg=tmpl&v=pref.html&text=yes">Preferences</a> . <a href="/cgi-bin/query?pg=q&text=yes">New Search</a> . <a href="/cgi-bin/query?pg=aq&text=yes">Advanced Search</a><br><BR><font size=-1 face="arial, helvetica"><b>3</b> documents match your query.</font><BR><dl>
<dt><font size=-1><b>1. </b></font><a href="http://www.engelschall.com/e/perldoc/pages/util/AutoSearch.html"><strong>AutoSearch -- a web-searching application</strong></a><dd>AutoSearch. NAME. SYNOPSIS. DESCRIPTION. OPTIONS. DESCRIPTION. AUTOMATED SEARCHING. CHANGING THE LOOK OF THE PAGES. FILES. SEE ALSO. AUTHORS. COPYRIGHT....<br><cite><a href="http://www.engelschall.com/e/perldoc/pages/util/AutoSearch.html">http://www.engelschall.com/e/perldoc/pages/util/AutoSearch.html</a> - size 16K - 29-May-97 - English</cite><br>
<P><dt><font size=-1><b>2. </b></font><a href="http://www.isi.edu/lsam/autosearch/TEST/index.html"><strong>Test Pages for AutoSearch</strong></a><dd>Test Pages for AutoSearch. Singleton. Dozen: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12. hidden anchor page. AutoSearch-page. Page maintainer: John Heidemann...<br><cite><a href="http://www.isi.edu/lsam/autosearch/TEST/index.html">http://www.isi.edu/lsam/autosearch/TEST/index.html</a> - size 1K - 5-Aug-97 - English</cite><br>
<P><dt><font size=-1><b>3. </b></font><a href="http://gost.isi.edu/~johnh/WORK/"><strong>John Heidemann's Work</strong></a><dd>John Heidemann's Work. I am computer scientist at Information Sciences Institute of the University of Southern California. I will also be teaching at USC..<br><cite><a href="http://gost.isi.edu/~johnh/WORK/">http://gost.isi.edu/~johnh/WORK/</a> - size 5K - 1-Apr-97 - English</cite><br>
<P></dl>
<b>Tip: </b>

To exclude a word, add a minus sign directly in front of it:
<b>eclipse -sun</b>
<br><BR><font size=-1 face="arial, helvetica"><font size=-1>Word count: LSAM: 92; AutoSearch: 369<BR></font>
</font><P><br>
<HR><font face="arial, helvetica" size=-1><a href="/av/content/av_network.htm">Our Network</a> | <a href="/av/content/addurl.htm">Add/Remove URL</a> | <a href="/av/content/questions.htm">Feedback</a> | <a href="/av/content/help.htm">Help</a><br><a href="http://www.doubleclick.net/jumps/jump1217.htm">Advertising Info</a> | <a href="/av/content/about.htm">About AltaVista</a> | <a href="/av/content/jobs.htm">Jobs</a> | <a href="/cgi-bin/query?pg=q">Graphics</a></font><HR><a href="/av/content/disclaimer.htm">Disclaimer</a> | <a href="/av/content/privacy.htm">Privacy Statement</a><br><a href="http://www.digital.com"><b>DIGITAL</b></a> Copyright 1997. All Rights Reserved</FORM>
</BODY></HTML>
