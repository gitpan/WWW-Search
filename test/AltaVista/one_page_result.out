http://www.engelschall.com/e/perldoc/pages/util/AutoSearch.html
http://www.isi.edu/lsam/autosearch/TEST/index.html
http://gost.isi.edu/~johnh/WORK/
